config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -142,6 +142,7 @@ presubmits:
     branches:
     - master
     always_run: false
+    cluster: k8s-infra-prow-build
```

Comment:
this cluster is for required presubmits and release blocking jobs currently.

Reply:
what should I use then?

Reply:
this is optional

Reply:
you can just drop this field, it will use the default cluster (`cluster: default`)